### PR TITLE
transforming project into single page

### PR DESCRIPTION
**Pull request by Arturgouveia1970**
we used Js to make project into single page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,22 +10,48 @@
...
-    <section>
-        <h1>Awesome Books</h1>
+    <nav>
+        <h2>Awesome Books</h2>
+        <ul>
+            <li><a id="list-link" href="#">List</a></li>
+            <li><a id="add-link" href="#">Add new</a></li>
+            <li><a id ="contact-link" href="#">Contact</a></li>
+        </ul>
+    </nav>
```
Comment (Cathella) [inline index.html:20]: ✅ It has a Navigation bar that navigates great 👍🏻.
✅ When any of the links are clicked, the main area changes to display only one of the three content sections 👍🏻.

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🟡 Status: Changes Required 🛑

### Hi @Arturgouveia1970, @bizip 👋🏻, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## To Highlight 👏🏻

✅ No linter errors.
✅ You used correct GitHub Flow.
✅ Has the three content sections - Books list, Add book form and Contact info.
✅ Used no library or framework to change the content of the page

## Required Changes ♻️
_Check the comments under the review._

- [ ] [REQUIRED] When I load the page the first time or even refresh it, all the sections are loaded which shouldn't be the case. Kindly fix the issue by hiding the sections that shouldn't show.
- [ ] Kindly edit the live project link in the README because it redirects to a completely different project.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by JuliCarracedo — CHANGES_REQUESTED** [review]
Hi @bizip  , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Professional README
✔️ Great job applying the required changes!

 ## Required Changes ♻️
_Check the comments under the review._

- [ ] The link to your live demo is still taking us to your portfolio project instead of taking us to Awesome books. Kindly fix this

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
**Hi @Team**,

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Arturgouveia1970** [comment]
Hi Julian,This is Artur, I am Bizimungu's partner. He has been sick in hospital since yesterday and I can't add alink do the project because we have been working on his repository. Can you please advise me on how to proceed? Thank you in advanceArtur Enviado do meu telemóvel Huawei.-------- Mensagem original --------De: Julián Carracedo ***@***.***>Data: quinta-feira, 28 de abril de 2022, 20:47Para: bizip/awesome-books ***@***.***>Cc: Artur Gouveia ***@***.***>, Mention ***@***.***>Assunto: Re: [bizip/awesome-books] transforming project into single page (PR #4)

@JuliCarracedo requested changes on this pull request.



Hi @bizip  ,

Great work on making the changes requested by a previous reviewer 👏

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Highlights

✔️ No linter errors

✔️ Professional README

✔️ Great job applying the required changes!

Required Changes ♻️

Check the comments under the review.

 The link to your live demo is still taking us to your portfolio project instead of taking us to Awesome books. Kindly fix this

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.



As described in the Code reviews limits policy  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

**Pull request merged**